src/category_theory/epi_mono.lean

```diff
@@ -48,5 +52,47 @@ lemma faithful_reflects_mono (F : C ⥤ D) [faithful F] {X Y : C} {f : X ⟶ Y}
   (hf : mono (F.map f)) : mono f :=
 ⟨λ Z g h H, F.injectivity $
   by rw [←cancel_mono (F.map f), ←F.map_comp, ←F.map_comp, H]⟩
+end
+
+class split_mono {X Y : C} (f : X ⟶ Y) :=
+(retraction : Y ⟶ X)
+(id' : f ≫ retraction = 𝟙 X . obviously)
+
+class split_epi {X Y : C} (f : X ⟶ Y) :=
+(section_ : Y ⟶ X)
```

Comment:
What is the reason for the underscore in `section_`? I guess it is because `section` is a reserved word?

Reply:
Could you please add a docstring? Probably you should mention relation to `mono`/`epi` and `is_iso`.

Reply:
Yes. Another alternative is to use "french quotes". I'm not sure which is worse, but of course I should add a comment explaining this.

Reply:
Added some comments about this.